src/lib/common/errors/error-500.svelte

```diff
@@ -18,7 +18,7 @@
 			</h1>
 			<h4 class="text-uppercase">Internal Server Error</h4>
 			<div class="mt-5 text-center">
-			  <Link class="btn btn-primary" href="/dashboard">
+			  <Link class="btn btn-primary" href="/page/dashboard">
```

Comment:
Can you change to ‘page/dashboard’ without root path.

Reply:
ok, I will fix it tomorrow and test it

Reply:
> Can you change to ‘page/dashboard’ without root path.

I have removed the root path and test it. 

I found another issue that the logos, in login and forget password page, have links to Dashboard page. I think these two links should be removed. (I didn't remove them in this commit)